src/main/java/org/commonjava/indy/service/tracking/client/content/ContentService.java

```diff
@@ -34,4 +36,8 @@
     @Path( "/tracking/recalculate" )
     Response recalculateEntrySet( final Set<ContentTransferDTO> entries );
 
+    @POST
+    @Path( "/repo/zip" )
+    File getZipRepository( final TrackedContent record );
```

Comment:
As there are some changes in indy content admin service for this API param, should we change it here accordingly?

Reply:
oh yes! updated it to use slimmer DTOs as in indy monolith